code/game/jobs/job_controller.dm

```diff
@@ -523,15 +523,18 @@ var/global/datum/controller/occupations/job_master
 							// This is a miserable way to fix the loadout overwrite bug, but the alternative requires
 							// adding an arg to a bunch of different procs. Will look into it after this merge. ~ Z
 							var/metadata = H.client.prefs.gear[G.display_name]
+							var/obj/item/CI = G.spawn_item(H,metadata)

```

Comment:
This may cause issues. As far as I can tell, it shouldn't spawn the item in certain cases.